test/e2e/testsuites/specs.go

```diff
@@ -251,3 +256,24 @@ func CreateVolumeSnapshotClass(client restclientset.Interface, namespace *v1.Nam
 
 	return tvsc, tvsc.Cleanup
 }
+
+func CreateVolumeSnapshotClassWithExternalResourceGroup(client restclientset.Interface, namespace *v1.Namespace, csiDriver driver.VolumeSnapshotTestDriver, creds *credentials.Credentials) (*TestVolumeSnapshotClass, func()) {
```

Comment:
what about reuse `CreateVolumeSnapshotClass` func and create resource group outside